apps/src/javalab/theater/Theater.js

```diff
@@ -46,14 +46,19 @@ export default class Theater {
         this.openPhotoPrompter(data.detail.prompt);
         break;
       }
+      case TheaterSignalType.NO_AUDIO: {
+        // there is no audio associated with the video, trigger startPlayback so we don't wait for the audio file
+        this.startPlayback();
```

Comment:
Do we hit any issues calling `play` on an audio element that has no `src`?

Reply:
oh good point. It does throw a console error, I'll clean this up.